l5kit/l5kit/dataset/chop_dataset.py

```diff
@@ -0,0 +1,96 @@
+import argparse
+from pathlib import Path
+
+import numpy as np
+from zarr import convenience
+
+from l5kit.data import ChunkedDataset
+from l5kit.data.zarr_utils import zarr_scenes_chop
+from l5kit.dataset.select_agents import TH_DISTANCE_AV, TH_EXTENT_RATIO, TH_YAW_DEGREE, select_agents
+from l5kit.evaluation import export_zarr_to_csv
+
+MIN_FUTURE_STEPS = 10
+
+
+def create_chopped_dataset(
+    zarr_path: str, th_agent_prob: float, num_frames_to_copy: int, num_frames_gt: int, min_frame_future: int
+) -> str:
+    """
+    Create a chopped version of the zarr that can be used as a test set. This function is used to generate tests
```

Comment:
still: tests set -> test sets

Reply:
Everything is singular now in that sentence. Do you want everything plural?

Reply:
For me the second sentence still is 'This function is used to generate tests set that ...'

Reply:
hahah sorry I thought you were referring to the first one :)